reference/create-index-using-pgroonga.md

```diff
@@ -148,6 +148,29 @@ CREATE INDEX pgroonga_tag_index
 
 See [Tokenizers][groonga-tokenizers] for other tokenizers.
 
+#### **Partial match in alphabetic languages** {#partial-match-alphabetic-languages}
+
+If you plan to perform partial matching searches for keywords in alphabetic languages, it is recommended to configure your tokenizer to `TokenNgram` with extra options. The default tokenizer in `PGroonga` is `TokenBigram`, which means that if you search for the keyword 'pp', for instance, it won't match 'Apple', 'Pineapple', or 'Ripple' in your data. To avoid this issue, it is strongly advised to set up your tokenizer as following `TokenNgram` example.
+
+Here is an example to use `TokenNgram` based tokenizer. You need to specify `tokenizer='TokenNgram'`. See [`TokenNgram`][groonga-token-ngram] for more detail.
+
+```sql
+CREATE TABLE memos (
+  id integer,
+  content text
+);
+
+CREATE INDEX pgroonga_content_index
+          ON memos
+       USING pgroonga (content)
+        WITH(tokenizer='TokenNgram("unify_alphabet", false, "unify_symbol", false, "unify_digit", false)');
+```
+
+You may also use `TokenBigramBigramSplitSymbolAlphaDigit` for partial match instead of `TokenNgram` above **(the former is recommended)**.
```

Comment:
Which is "the former" in this case?

How about writing "`TokenNgram(...)` is recommended" explicitly?

Reply:
Good point! I made it explicit!